### ISSUE TITLE
Would be really nice to reprocess prep if you make a mistake without reloading and unlinking data

### ISSUE THREAD

**Issue by ackermag** [biocore/qiita#1569]
e.g. select 16S, should be 18S;
- select the wrong preprocess option e.g. 16S, should be 18S

Would be much cleaner if prep-template is fine and data is fine to just reprocess with different options (rev comp) or for different target gene (16S, 18S, ITS)

**Comment by antgonza** [comment]
@ackermag I think there are 2 issues here:
1. be able to redefine the data type in a prep info file, which should be propagated to all artifacts from that prep info file 
2. be able to reprocess the same prep-info/raw-data using different parameters: this is currently possible

If you agree, I'll change the issue name.

**Comment by josenavas** [comment]
I think this goes back to the fact that from the system point of view there is no difference between 16S, 18S or ITS - it doesn't affect anything on the processing pipeline. We should allow changing that as long as they are under the same type (i.e. we can change the description to `target_gene.{16S,18S,ITS}` and allow to change it as long as is in the target_gene.X series).

**Comment by antgonza** [comment]
Right, do we have that definition somewhere or is that something we have to do? If we need to do, should that be a different issue?

**Comment by antgonza** [comment]
@josenavas, ping! What do you think?

**Comment by antgonza** [comment]
Renaming the issue for clarity.

**Comment by josenavas** [comment]
How do we want to handle this? To be fair, the plugins don't even care about it, as it is applied in a per artifact basis. However, we can wrap this information as part of the type decorators that we were talking so plugins are aware of them, but let's revisit this after tomorrow's meeting.

**Comment by antgonza** [comment]
Possible. Let's discuss during the Qiita meeting. Moving to Alpha 0.4.
